src/Main/App.js

```diff
@@ -1,32 +1,26 @@
-import React, { useState, useEffect } from "react";
-import "./App.css";
+import React, { useState, useEffect, useContext } from "react";
+import "./App.scss";
 import Frontpage from "../Pages/Frontpage/Frontpage";
 import Dashboard from "../Pages/Dashboard/Dashboard";
-import { UserProvider } from "./UserContext";
+import { UserContext } from "./UserContext";
+import Header from "../Shared/Header";
+import Footer from "../Shared/Footer";
 
-// Dependencies
-// npm install reactjs-popup
-// https://www.npmjs.com/package/reactjs-popup
-
-// npm install yup
-// https://www.npmjs.com/package/yup
-
-// npm install react-hook-form
-// npm install @hookform/resolvers
-// https://react-hook-form.com/api/
-
-// npm install js-yaml
-// npm install fs
 function App() {
-  // const [loggedIn, setLoggedIn] = useState("");
+  const loggedIn = useContext(UserContext).loggedIn[0]
   const [display, setDisplay] = useState();
 
   useEffect(() => {
     setDisplay(<Frontpage setDisplay={setDisplay} />);
-    // setDisplay(<Dashboard />);
   }, []);
 
-  return <UserProvider className="App">{display}</UserProvider>;
+  return (
+    <div className="App">
+      <Header />
+      {loggedIn ? <Dashboard /> : display}
+      <Footer />
+    </div>
+  );
 }
```

Comment:
Yeah. We are sorta working on that. Debating whether we have the time atm. It will probably be added closer to the final handin deadline.

Reply:
Mads do you think its worth the effort refactoring to use router at this point?? We talked about it and at this point it seems like it would require a mayor refactoring for little improvement?

Reply:
Guess it depends on what else you need to work on - you might be able to make it without, but could also end up spending a lot of time trying to make "work around" this problem. 

But yeah, it could make sense to postpone this since it's kinda working now, and then focus on your requirements first - then later if you have time/need for it you can do this :)